tests/js/handler/handler-caseholder.js

```diff
@@ -0,0 +1,71 @@
+/* Tests for the "express" and "router" module */
+"use strict";
+var fluid        = fluid || require("infusion");
```

Comment:
Simply var fluid = require("infusion") is correct here since you are running in node.js